src/hotspot/share/opto/c2compiler.cpp

```diff
@@ -64,7 +64,7 @@ bool C2Compiler::init_c2_runtime() {
       OptoReg::vm2opto[i] = OptoReg::Bad;
   }
 
-  for( OptoReg::Name i=OptoReg::Name(0); i<OptoReg::Name(REG_COUNT); i = OptoReg::add(i,1) ) {
+  for (OptoReg::Name i=OptoReg::Name(0); i<OptoReg::Name(REG_COUNT); i = OptoReg::add(i,1)) {
```

Comment:
I don't think it's worth it to change this line just because of white space.

Reply:
Removed. Thanks for the prompt review!